### DISCUSSION TITLE
Is there a way to swap snap behavior?

### DISCUSSION THREAD

**Discussion by Lukeout** [rxhanson/Rectangle#281] [Q&A]
The way my monitor works, I want to use thirds but sometimes its buggy when trying to drag to bottom. Can I swap top and bottom snapping behavior; i.e so that top left is 1/3 left, and so on, etc

**Comment by rxhanson** [comment]
The snap areas are not currently configurable, but it's something that I've wanted to add in (or would accept a pull request for). 

if you don't mind me asking, is the buggy behavior when trying to drag to the bottom something that could be fixed in Rectangle?

**Comment by Lukeout** [comment]
Good question, yeah I've found it to be less reliable then drag on top. Esp the middle section as my dock is there. Note im on a 49inch monitor so I'm using thirds

**Comment by rxhanson** [comment]
Try v0.39. I've put in a bug fix for the bottom snap areas being unreliable. 

https://github.com/rxhanson/Rectangle/releases/tag/v0.39
